package.json

```diff
@@ -15,7 +15,7 @@
     "test": "mocha --timeout 0 -r ts-node/register generators/test/**.js"
   },
   "engines": {
-    "node": ">6.9.0"
+    "node": ">=12"
```

Comment:
Can we make this node 14? Which is the current LTS version.

Reply:
Updated to v14